scipy/optimize/_qap.py

```diff
@@ -271,7 +271,7 @@ def _quadratic_assignment_faq(A, B,
         \mbox{s.t. } & {P \ \epsilon \ \mathcal{P}}\\
 
     where :math:`\mathcal{P}` is the set of all permutation matrices,
-    and :math:`A` and :math:`B` are square matrices with non-negative elements.
+    and :math:`A` and :math:`B` are square matrices.
```

Comment:
There are 7 other instances of "non-negative" in the document.

Reply:
You're right, I'll fix that